src/com/oltpbenchmark/api/Worker.java

```diff
@@ -562,6 +569,8 @@ protected final TransactionExecutionState doWork(boolean measure, SubmittedProce
                     // ------------------
                     } else if (ex.getErrorCode() == 0 && ex.getSQLState() != null && ex.getSQLState().equals("40001")) {
                         // Postgres serialization
+                        totalRetries++;
+                        status = TransactionStatus.RETRY;
```

Comment:
looks like there are other situations where we maybe want to retry, e.g. L587, based on the comment

Reply:
I didn't want to retry unless we have transaction conflicts cuz they should be the most common failures that we should retry. Other failures can be ignored now that we don't count failures now as successes.